pkg/istio/client/pilotv2.go

```diff
@@ -0,0 +1,99 @@
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	envoy_api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+	envoy_api_core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	xds "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
+	"github.com/go-mesh/mesher/pkg/istio/util"
+	"google.golang.org/grpc"
+)
+
+// PilotClient is a interface for the client to communicate to pilot
+type PilotClient interface {
```

Comment:
can you use unified client in pkg/infras/istio

Reply:
addressed the review comment